### PR TITLE
send crowdfund contributions more often + refund crowdfund contributions when a bounty is canceled

### PR DESCRIPTION
**Pull request by owocki**
##### Description

send crowdfund contributions more often + refund crowdfund contributions when a bounty is canceled. 

this was discussed on the ethereal hackathon call today.  

##### Refers/Fixes

ethereal hackathon call today

##### Testing

tested it locally

### CODE REVIEW

File: app/dashboard/management/commands/send_tips_for_bounty_fulfiller.py [hunk 1]
```diff
@@ -56,59 +56,22 @@ def handle(self, *args, **options):
...
+                        # send to fulfiller
+                        ######################################################
+                        print(" - 1 ")
+                        # assign tip to fulfiller and email them
+                        tip = assign_tip_to(tip, fulfillment.fulfiller_github_username)
+                        msg = f'auto assigneed on {timezone.now()} to fulfillment {fulfillment.pk}; as done bountyfulfillment'
+                        print("     " + msg)
+                        tip.metadata['payout_comments'] = msg
+                        tip.save()
+                    elif send_back_to_where_it_came:
```
Comment (octavioamu) [inline app/dashboard/management/commands/send_tips_for_bounty_fulfiller.py]: lol that name

File: app/dashboard/management/commands/send_tips_for_bounty_fulfiller.py [hunk 2]
```diff
@@ -56,59 +56,22 @@ def handle(self, *args, **options):
...
-                                  'bounty w no bountyfulfillment'
-                            print("     ", msg)
-                            # TODO: email recipients of the cloned tip
-                            tip.metadata['payout_comments'] = msg
-                            tip.save()
-                    elif bounty.status == 'cancelled':
+                    fulfillment = bounty.fulfillments.filter(accepted=True)
+                    send_to_fulfilled_parties = bounty.status == 'done' and fulfillment.exists()
+                    send_back_to_where_it_came = bounty.status == 'cancelled' or (bounty.status == 'done' and not fulfillment.exists())
+                    if send_to_fulfilled_parties:
```
Comment (octavioamu) [inline app/dashboard/management/commands/send_tips_for_bounty_fulfiller.py]: should we respect the % or keep splitting in equals parts? 
I was thinking today a way to know how much is the % to pay could be taking the amount payed for that user / 100 * total

Comment (owocki) [reply app/dashboard/management/commands/send_tips_for_bounty_fulfiller.py]: > should we respect the % or keep splitting in equals parts?

defer to @PixelantDesign @frankchen07 @ceresstation on this

Comment (frankchen07) [reply app/dashboard/management/commands/send_tips_for_bounty_fulfiller.py]: The question I would ask is, does crowdfunding in this case mean you're putting money in the "hands of the funder" or the "hands of the contributor"? 

If its the former, then the crowdfunder is trusting that the funder will review accordingly and use his/her additional reward appropriately. If its the latter, then the crowdfunder is hoping that their contribution goes to the contributor, and they have less information about how it should be rewarded (unless they want to review the PR).

My thinking is we honor the % that the funder sets.

**Comment by thelostone-mc** [comment]
Closing this out as we've blocked this via the UI!

**Pull request closed without merging**